main/mariadb/APKBUILD

```diff
@@ -315,6 +320,15 @@ _server_utils() {
 }
 
 sha512sums="7702af8787973b15784dc8a89f9d56765725a420e2e68f1924b67c88c134c7d83d3249dddd16707fa9e5f05ed747f813f16feefbb477e999a6698b1dcfb5a88f  mariadb-10.2.15.tar.gz
+_plugin_rocksdb() {
```

Comment:
the `sha512sum` line needs to be moved below the `_plugin_rocksdb()` function. It looks like it was pasted wrong location.